clients/cobol-lsp-vscode-extension/src/services/util/Utils.ts

```diff
@@ -49,11 +49,11 @@ export async function getExtensionApi<T>(
   }
   return {
     futureApi: new Promise((res, _) => {
-      const extAdded = vscode.extensions.onDidChange(() => {
+      const extAdded = vscode.extensions.onDidChange(async () => {
         const ext = vscode.extensions.getExtension(extName);
         if (!ext) return;
         extAdded.dispose();
-        extractApi<T>(ext, validate).then((api) => res(asAPI<T>(api)));
+        await extractApi<T>(ext, validate).then((api) => res(asAPI<T>(api)));
```

Comment:
This `await` does not really make much sense to me. It is supposed to resolve the `futureApi` promise.